### ISSUE TITLE
HW1: javascript - email description

### ISSUE THREAD

**Issue by jinsun1117** [swsnu/swppfall2019#15]
From the README specification

> Email: characters@characters.domain (characters other than @ or whitespace followed by an @ sign, followed by more characters (not '@', '.', or whitespace: co.kr is not allowed in this case), and then a ".". After the ".", you can only write 2 to 3 letters from a to z).

The part 

> After the ".", you can only write 2 to 3 letters from a to z

directly collides with the email example (@characters.domain), since 'domain' consists of 6 letters(rather than 2 to 3 letters as written in the specification.

Naturally, I guess the intent of the homewok would be to stick with '2 to 3 letters' and that the '@characters.domain' is not a valid example. However, I would like to confirm my thoughts. 

Thank you.

===
예시문에 보면 valid 한 예시로 @characters.domain 이 제시되어 있습니다만, 스펙상에는 . 뒤에는 '2 ~3 개 글자'만 허용하고 있어 충돌이 있는것같아 보입니다. 당연히 스펙(2~3글자)을 따르면 될것으로 예상하지만, 혹시나 싶어 문의드립니다.

감사합니다.

**Comment by ktaebum** [comment]
1. Yes, please follow the specification (2 to 3 letters). `.domain` is just for example, and maybe `.com ` is a more proper example in real account (like `id@google.com`)

2. Yes you must fill constructor too

**Comment by zeroFruit** [comment]
@ktaebum 

![image](https://user-images.githubusercontent.com/18715362/64551071-3d9c6100-d36f-11e9-8181-e406e8aa9460.png)


I have question about 'email description'!

On the example of `characters@characters.domain`, you specified `domain` part as a-z but not about `characters` part. `characters` part also allow lowercase only?

----

email description과 관련이 있어보여 이어서 씁니다!

`characters@characters.domain` 에서 `domain` 부분은 a-z로 명시를 해주셨는데 `characters` 부분은 대소문자와 관련해서 명시가 안된거 같아서 질문드립니다. `characters` 부분도 마찬가지로 소문자만 허용하나요?

**Comment by hy00nc** [comment]
@zeroFruit 
characters include all the alphabet even the uppercase one, and the same for domain part (a-z includes both uppercase and lowercase). Sorry for the confusion.

**Comment by wormfry** [comment]
I'm confused about what 'characters' exactly means. I think  0-9 could be considered as characters, and '#','%' signs could be considered as characters. What is the exact range of 'characters'?

**Comment by hy00nc** [comment]
@wormfry You're right, *characters* include **everything** we can type including special characters and even space.

**Comment by sylee0124** [comment]
not sure if i got this right but whitespace followed by an @ means that 
 @naver.com is ok?

제가 잘 이해했는지 모르겠는데요 그 whitespace followed by an @은 허용해야한다는게
 @naver.com 이 가능하면 되는거죠?

**Comment by hy00nc** [comment]
`characters other than @ or whitespace followed by an @ sign` <- this means before @, no @ or whitespace can come. So `'/s'@naver.com` is not allowed.

@ 전에는 공백이나 또 다른 @이 오면 안 된다는 의미입니다!

**Comment by sylee0124** [comment]
그리고 characters이라고 했으면 t@naver.com이나 @naver.com은 불가능한건가요?

**Comment by hy00nc** [comment]
@sylee0124 `characters` means one or more characters so `t@naver.com` is correct. `@naver.com` is not allowed of course, since there is no character(s) before @.

**Comment by sylee0124** [comment]
답변해주셔서 감사합니다 : )
